Azure/Prod/resource_sql.tf

```diff
@@ -258,3 +266,107 @@ DEPLOY
 
   deployment_mode = "Incremental"
 }
+
+# New SQL Database using for the BuyingCatalogueService Private
+resource "azurerm_sql_database" "sql-bc-bapi-pri" {
+  name                             = "bc-buyingcatalogue-private-bapi"
+  resource_group_name              = azurerm_resource_group.bc-sql-pri.name
+  location                         = var.region
+  server_name                      = azurerm_sql_server.bc-sql-pri.name
+  collation                        = var.sql_collation
+  edition                          = var.sql_edition
+  requested_service_objective_name = "S0" #var.sql_size
```

Comment:
I think these are S1 - check the existing DBs, they should be the same (they have less spec in test)

Reply:
Ah ok, I was mirroring Test (where the new DBs in test are lower spec than the current ones). I'll fix. Cheers

Reply:
Prod updated to use Variable (S1) as requested